### DISCUSSION TITLE
Linux mint install script

### DISCUSSION THREAD

**Discussion by max246** [Chia-Network/chia-blockchain#6099] [Ideas]
I have Linux mint and I use the ubuntu guide to setup the system.

Would be good to avoid to install NPM via apt-get since nodejs is installed and the latest version doesnt need install npm as separate package as it is part of the nodejs package.

```
The following packages have unmet dependencies.
 nodejs : Conflicts: npm
 npm : Depends: node-gyp (>= 3.6.2~) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

I just bypass this error by editing the script install-gui.sh and removing NPM, but would be great if you can add a flag or second script that doesnt require to install npm since it is part of nodejs package.

**Comment by laz954** [comment]
I am having the same issue with all xforks can you share how you commented this out ? I accidentally deleted some file and have hot been able to get any of the xforks to work since, going on 1 week now and I haven't seen any other comments on this. email: lazaroperez@comcast.net Thanks in advance.
